features/step_definitions/active_docs_steps.rb

```diff
@@ -1,48 +1,60 @@
 # frozen_string_literal: true
 
-Given(/^the provider has (\d) active docs?$/) do |number_active_docs|
-  @active_docs = FactoryBot.create_list(:api_docs_service, number_active_docs.to_i, account: @provider, service: @provider.default_service)
+Given "the provider has {int} active doc(s)" do |number_active_docs|
+  @active_docs = FactoryBot.create_list(:api_docs_service, number_active_docs, account: @provider, service: @provider.default_service)
 end
 
-When /^I try to (create|update) the active docs( of the service)? with (in)?valid data$/ do |action, optional_scope, invalid|
+When "I try to {word} the active docs with {valid} data" do |action, valid|
```

Comment:
Original step makes more obvious what values `word` may have. Do we need to convert everything to cucumber expressions? Or can we use a custom type or maybe validation inside the step? Or maybe not a problem for anybody else, in which case, keep as is.

P.S. same with next step.